windows-apps-src/design/style/reveal.md

```diff
@@ -134,15 +134,30 @@ These effects are defined by two brushes:
 In most cases we handle the usage of both of them by turning Reveal on automatically for a certain controls. However, other controls will need to be enabled through applying a style, or changing their templates directly.
 
 ### When to add Reveal
-You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves. 
-* If your custom control is a single interactive element and doesn't have similar controls sharing it's space (such as menu items in a menu), it's likely that your custom control doesn't need Reveal.  
-* If you have a grouping of related interactive content or elements, then it's likely that that region of your app does need Reveal - this is commonly referred to as a [Commanding](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding) surface.
-
-For example, a button by itself shouldn't use reveal, but a set of buttons in a command bar should use Reveal.
+You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves.
+* **Does your custom control sit a very UI dense area?**
+Chances are you have need of Reveal - as Reveal works great in scenarios where you want to define control state, and interaction type in a UI dense area.
+Examples of this can be seen in the semantic zoom area in the start menu:
+![Reveal Semantic](images/StartMenuSemantic_Reveal.png)
+And on command bars, like the one in the Photos app:
+![Reveal Semantic](images/CommandBar_Reveal.png)
+
+* **Is your control sitting in a group or list of other similarly sized, interactive items?**
+Then you probably want Reveal on all of them to help the interactivity of each stand out.
+A good example of situations like these are the tiles on the Start menu:
+![Reveal Tiles](images/StartMenuTiles_Reveal.png)
+
+* **Is your control placed far away from other interactive elements?**
+Reveal actually is not the best choice for these scenarios.
+Calculator is a great example of this:
+![Reveal Calculator](images/HamburgerButtonCalculator_Reveal.png)
+The hamburger button in the upper left is not crucial to their app's overall experience, not to mention it's not near anything else - and therefore it doesn't need any Reveal enhancements.
+
+In general, Reveal highlight brings attention to certain areas of your app, or helps define interaction in very UI dense surfaces. Consider your control's neighbors, do they have a lot of interactive items nearby? Then you might want to consider adding Reveal.
```

Comment:
@chigy - can you not see these updates?

Reply:
I'm sure it is user error but I don't see images. They show up as links and it goes to 404 page if I clicked.